docs/getting-started/technical-overview.md

```diff
@@ -0,0 +1,25 @@
+# Technical overview
+
+## The core tech
+
+In brief, the main technologies behind Apostrophe are:
+
+- [Node.js](https://nodejs.org): **Popular server-side JavaScript runtime.** We originally chose Node.js to achieve a fully-Javascript development experience for developers. It has continued to improve over time, delivering high performance and powerful features we can use.
+- [Express](https://npmjs.org/package/express): **Un-opinionated Node.js web framework.** Express is the most widely used web framework for Node.js. Because it is simple, un-opinionated, and well-known, it was straightforward to extend it to suit Apostrophe's needs.
+- [MongoDB](https://www.mongodb.com): **Secure, document-based database engine.** We chose MongoDB for its fluent JavaScript-based API, its safety from "SQL injection"—style attacks, its developer-friendliness, and its support for documents with varying schemas in the same collection.
+- [Nunjucks](https://mozilla.github.io/nunjucks/): **Richly featured template language for Javascript.** Nunjucks provides tons of features, extensibility, and a syntax nearly identical to Twig, Jinja, and other Django-inspired templating languages.
+
+## Directory structure
+
+There are a few directories and top-level files that are especially important in Apostrophe projects. Here is a rough overview of these files and folders you will see in the official boilerplate.
+
+| Folder/File | What is it? |
+| ------ | ------ |
+| `app.js` | The heart of the application. This is where you do some general configuration and tell Apostrophe what modules are in your project. |
```

Comment:
This is a bit clunky. I'm trying to think on how to remove "**this is where you do**" some general configuration (what kind?) and **tell** apostrophe.

Reply:
How about:

"This is where you tell Apostrophe what modules are in your project and set a few top-level parameters."